lldb/bindings/interface/SBFrameDocstrings.i

```diff
@@ -78,6 +78,22 @@ See also SBThread."
     See also GetFunctionName()."
 ) lldb::SBFrame::IsInlined;
 
+%feature("docstring", "
+    Returns an SBValueList which is an array of one or more register
+    sets that exist for this thread.  
+    Each SBValue in the SBValueList represents one register-set. 
+    The first register-set will be the general purpose registers -- 
+    the registers printed by the `register read` command-line lldb, with 
```

Comment:
```suggestion
    the registers printed by the `register read` command-line in lldb, with 
```

Reply:
I think in my head this started as "the registers printed by the `register read` command in command-line lldb" and it got a little mangled somewhere along the way.